ebl-issuance/src/main/java/org/dcsa/conformance/standards/eblissuance/party/EblIssuanceCarrier.java

```diff
@@ -53,11 +53,35 @@ public EblIssuanceCarrier(
     this.payloadSigner = payloadSigner;
   }
 
+  private static byte[] generateDocument() {
+    byte[] pdf;
+    String filepath = "/standards/eblissuance/messages/test-iss-document.pdf";
+    try (InputStream inputStream = EblIssuanceCarrier.class.getResourceAsStream(filepath)) {
+      if (inputStream == null) {
+        throw new IllegalArgumentException("File not found: " + filepath);
+      }
+      pdf = inputStream.readAllBytes();
+    } catch (IOException e) {
+      throw new IllegalArgumentException(
+          "Generating document failed. Could not load file: " + filepath);
+    }
+    String uuidHex = UUID.randomUUID().toString();
+    String pdfString = new String(pdf, StandardCharsets.ISO_8859_1);
+    String title = "/Title (DCSA - Driving digitalisation in container shipping)";
+    int titleStart = pdfString.indexOf(title);
+
+    int titleEnd = pdfString.indexOf(")", titleStart);
+    String newTitle = "DCSA - " + uuidHex + " shipping";
+
+    String updatedPdfContent =
+        pdfString.substring(0, titleStart + 7) + newTitle + ")" + pdfString.substring(titleEnd + 1);
+    return updatedPdfContent.getBytes(StandardCharsets.ISO_8859_1);
+  }
```

Comment:
Interesting to see that reformatting the code also shifts methods to different places 🤔 
I don't mind, but it is not required/needed.